src/orange/orangeinp/IntersectRegion.cc

```diff
@@ -1177,6 +1145,51 @@ void Parallelepiped::output(JsonPimpl* j) const
     to_json_pimpl(j, *this);
 }
 
+//---------------------------------------------------------------------------//
+// PLANEALIGNED
+//---------------------------------------------------------------------------//
+/*!
+ * Construct with sense, axis, and position.
+ */
+PlaneAligned::PlaneAligned(Sense sense, Axis axis, real_type position)
```

Comment:
The top level doc string says that an `IntersectRegionInterface` is a "non-reentrant spatial [region]." Thus I think this should be renamed `Halfspace` or `AlignedHalfSpace`.

Reply:
`PlaneAlignedHalfspace`? Since `PlaneAligned<Axis>` is our templated class I wanted to keep the name consistent.

Reply:
(My previous message was sent before I saw your last one, due to being interrupted by a meeting)

`PlaneAlignedHalfspace` is fine. My main comment is that is confusing to have a file full or different convex regions and have one of them designed as a plane.

Reply:
Actually I am going for `InfPlane` analogous with `InfWedge` etc.

Reply:
Didn't see your messages either, actually, lol... I think that since they are all convex regions the concept is conveyed clearly? The semi-infinite ones (wedge, plane) are also convex.

Reply:
Just approved MR. I do think `Halfspace` is more descriptive than `Plane` but ultimately it doesn't matter too much.

Reply:
I think enough people think "halfspace" in the sense of _any_ quadric surface dividing the universe, so I see the term informally used to refer to surface plus sense.